website/common/locales/en/character.json

```diff
@@ -170,6 +170,8 @@
   "youCast": "You cast <%= spell %>.",
   "youCastTarget": "You cast <%= spell %> on <%= target %>.",
   "youCastParty": "You cast <%= spell %> for the party.",
+  "chatCastSpellParty": "<%= username %> casts <%= spellName %> for the party.",
+  "chatCastSpellUser": "<%= username %> casts <%= spellName %> on <%= target %>.",
```

Comment:
nit/suggestion: the `youCastTarget` and `youCastParty` strings above use the `spell` variable, so also use `spell` (not `spellName`) here for consistency